WordPress/Classes/ViewRelated/Gutenberg/Views/GutenGhostView.swift

```diff
@@ -93,12 +93,7 @@ class GutenGhostView: UIView {
 }
 
 private extension UIColor {
-    static var ghostToolbarBackground: UIColor {
-        if #available(iOS 13, *) {
-            return UIColor(light: .clear, dark: UIColor.colorFromHex("2e2e2e"))
-        }
-        return .clear
-    }
+    static var ghostToolbarBackground: UIColor = UIColor(light: .clear, dark: UIColor.colorFromHex("2e2e2e"))
```

Comment:
```suggestion
    static let ghostToolbarBackground: UIColor = UIColor(light: .clear, dark: UIColor.colorFromHex("2e2e2e"))
```

PS: I'm surprised that you can do that in an `extension` as you can't add instance variable / storage in `extension`… but not sure about add `static` ones on the type. If that's not allowed, I'd assume that restriction applies equally to `static let` and `static var` storages, and the solution would be to go back to a computed variable `static var ghostToolbarBackground: UIColor { .init(…) }` instead

Reply:
Yes, it is allowed.

I wouldn't be surprised if it was a recent Swift change though, because I remember running into the constraint of not being able to add storage in `extension` in the past. _Or, maybe I never tried to do it with `static`._

Reply:
> I wouldn't be surprised if it was a recent Swift change though, because I remember running into the constraint of not being able to add storage in extension in the past. _Or, maybe I never tried to do it with `static`._

Might actually be a case of me never running into this with `static` or always assuming it worked the same as with instance. There is [at least one StackOverflow answer](https://stackoverflow.com/a/33603246/809944) using `static left` in an extension, dated 2014.

Reply:
Oh, and by the way, stored properties are still not allowed:

<img width="1132" alt="image" src="https://user-images.githubusercontent.com/1218433/198162821-bd595c1c-83d1-42a3-a773-10cdb68ec1ef.png">

Reply:
Ah, yeah, that makes sense I guess.

I totally get why there is this limitation for `var` instance variables. But given that `let` are constants, and thus not taking storage space on the instance and thus not modifying the underlying memory layout of an instance of the type, I wonder why instance constants `let` are not allowed tbh (after all, they could be seen as synonyms to `var potato: String { "potato" }`, at least in behavior…).

[Maybe this will be allowed in a future version of Swift though](https://forums.swift.org/t/allow-stored-properties-in-extensions/60889)?